static/app/views/dashboards/manage/index.tsx

```diff
@@ -177,6 +180,20 @@ class ManageDashboards extends AsyncView<Props, State> {
           onChange={opt => this.handleSortChange(opt.value)}
           position="bottom-end"
         />
+        <Feature features={['dashboards-import']}>
+          <Wrapper>
+            <Button
+              onClick={() => {
+                openImportDashboardFromFileModal({organization, api, location});
+              }}
+              size="sm"
+              priority="primary"
+              icon={<IconAdd isCircled />}
+            >
+              {t('Import Dashboard from JSON')}
+            </Button>
+          </Wrapper>
+        </Feature>
```

Comment:
I noticed that the button's rendered underneath the search bar, if we move this code and drop the `Wrapper` (I don't think we need it) we can put it in the header bar next to the "Create Dashboard" button which makes it look a little less conspicuous. You would just move this to line 328

<img width="1484" alt="Screenshot 2023-07-06 at 1 06 08 PM" src="https://github.com/getsentry/sentry/assets/22846452/fd772374-7d7a-46ae-bff8-80ea4fbcf8d0">

Reply:
Good idea 👍